apps/condo/domains/notification/transports/push.js

```diff
@@ -16,15 +16,17 @@ const {
     PUSH_TRANSPORT_HUAWEI,
 } = require('@condo/domains/notification/constants/constants')
 const { renderTemplate } = require('@condo/domains/notification/templates')
-const { RemoteClient } = require('@condo/domains/notification/utils/serverSchema')
-
-const logger = getLogger('messaging/sendNotification')
 
 const ADAPTERS = {
     [PUSH_TRANSPORT_FIREBASE]: new FirebaseAdapter(),
     [PUSH_TRANSPORT_HUAWEI]: new HCMAdapter(),
     [PUSH_TRANSPORT_APPLE]: new AppleAdapter(),
 }
+const responseResolversByTransport = {
```

Comment:
Looks like bad remote client cleaners or something like that …

Reply:
If the application has not been used for a month, the token becomes expired. In essence, the user loses the authorization session in fcm. Since our architectural tokens are stored not in Redis, but in the db, we delete the outdated authorization session by clearing the field in the db.